@app/components/src/AvatarUpload.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from "react";
+import { Upload, Icon, message } from "antd";
+import { UploadChangeParam } from "antd/lib/upload";
+import { UploadFile, RcCustomRequestOptions } from "antd/lib/upload/interface";
+import axios from "axios";
+import {
+  useChangeAvatarMutation,
+  ProfileSettingsForm_UserFragment,
+} from "@app/graphql";
+import { ApolloError } from "apollo-client";
+
+export function slugify(string: string) {
```

Comment:
Instead of defining your own function to make a slug, it's probably better to use an existing package. I just searched through `yarn.lock`, and it looks like we already depends on a library called [`unique-slug`](https://github.com/npm/unique-slug), which would probably suit your needs just fine.